examples/log-generator/Dockerfile

```diff
@@ -0,0 +1,13 @@
+FROM node:8.2.1-alpine
+
+WORKDIR /log-generator
+
+ENV ESCRIBA_TIMEOUT 3000
```

Comment:
It's a default value, doesn't affect the code. It's just to show in Dockerfile: 'Hey we have an environment variable here'. The user input(-e option) and the default inside the code is the "master". Also if you want to use docker inspect to do some other task you will have the full metadata about this image.